wp-includes/widgets/class-wp-widget-media-video.php

```diff
@@ -59,16 +59,10 @@ public function get_instance_schema() {
 		$schema = array_merge(
 			parent::get_instance_schema(),
 			array(
-				'poster' => array(
-					'type' => 'string',
-					'default' => '',
-					'format' => 'uri',
-					'description' => __( 'URL to the poster frame' ),
-				),
 				'preload' => array(
 					'type' => 'string',
 					'enum' => array( 'none', 'auto', 'metadata' ),
-					'default' => 'none',
+					'default' => 'auto',
```

Comment:
Good catch. The default for video should actually be `metadata`, per https://github.com/WordPress/wordpress-develop/blob/9e07667864ca860d8683e30ccfdca796229c5657/src/wp-includes/media.php#L2463